### DISCUSSION TITLE
Hits@ K , MRR , AMRI

### DISCUSSION THREAD

**Discussion by baalagirish92** [pyg-team/pytorch_geometric#6908] [Q&A]
I would like to know if there is a way to calculate Hits @K, MRR and AMRI for Heterogeneous Data. Struggling to implement these Metrics and understand them. Would love to here ideas from you guys !!!

**Comment by cogmind** [comment]
M

**Comment by rusty1s** [comment]
I suggest to take a look at `torchmetrics` for an awesome library to help with metric computation :)
